checkstyle/import-control.xml

```diff
@@ -475,14 +477,15 @@
       <allow pkg="scala" />
       <allow class="kafka.zk.EmbeddedZookeeper"/>
       <allow pkg="com.fasterxml.jackson" />
+      <allow pkg="org.apache.kafka.tools" />
     </subpackage>
 
     <subpackage name="test">
       <allow pkg="kafka.admin" />
     </subpackage>
 
     <subpackage name="tools">
-      <allow pkg="kafka.tools" />
+      <allow pkg="org.apache.kafka.tools" />
```

Comment:
This subpackage is not needed if you move StreamsResetterTest to tools test.

Reply:
I guess this is related to my comment https://github.com/apache/kafka/pull/13127#discussion_r1104203628?